themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1580,29 +1580,24 @@ html body.tc-body.tc-single-tiddler-window {
 	overflow: auto;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div {
+.tc-grid {
 	display: grid;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div.tc-tiddler-preview {
-	grid-template-areas: 
-	"toolbar toolbar"
-	"editor preview";
+.tc-tiddler-frame .tc-grid.tc-tiddler-preview {
+	grid-template-areas: "toolbar toolbar" "editor preview";
 	grid-template-columns: 1fr 1fr;
 	grid-template-rows: auto 1fr;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div:not(.tc-tiddler-preview) {
-	grid-template-areas: 
-	"toolbar"
-	"editor";
+.tc-tiddler-frame .tc-grid.tc-no-tiddler-preview {
+	grid-template-areas: "toolbar" "editor";
```

Comment:
also here. IMO it's much easier to understand

```
	grid-template-areas:
		"toolbar"
		"editor";
```

Reply:
I made that more readable, too